workflow/executor/executor.go

```diff
@@ -263,6 +263,9 @@ func (we *WorkflowExecutor) StageFiles() error {
 		filePath = common.ExecutorScriptSourcePath
 		body = []byte(we.Template.Script.Source)
 	case wfv1.TemplateTypeResource:
+		if we.Template.Resource.ManifestPath != "" {
+			return nil
```

Comment:
`ManifestPath`. should be.optional.  it should not return. here

Reply:
I think it's optional. If `ManifestPath` not provided, it will not return.